/**/send_obstacles:
  ros__parameters:
    n_obstacles: 5
    no_cylinders: false
    no_boxes: false
    max_size: 2.5
    min_size: 1.0

/**/send_gates:
  ros__parameters:
    use_namespace: false
    # Values of x and y different from 0.0 will spawn the gates in the given position    
    x: 0.0
    y: 0.0 

/**/send_victims:
  ros__parameters:
    n_victims: 0
    min_weight: 10
    max_weight: 500

/**:
  ros__parameters:
    map: hexagon #hexagon, rectangle
    dx: 5.0
    dy: 10.0
    max_timeout: 3
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: transient_local
          history: keep_last
          reliability: reliable

